pkgs/applications/graphics/antimony/paths-fix.patch

```diff
@@ -1,99 +1,36 @@
-+QMAKE_CXXFLAGS += "-D'GITTAG=\"$$(GITTAG)\"'"
-+QMAKE_CXXFLAGS += "-D'GITBRANCH=\"$$(GITBRANCH)\"'"
- 
- OLD_GL_SET = $$(OLD_GL)
- equals(OLD_GL_SET, "true") {
-@@ -125,11 +120,11 @@ macx {
- }
- 
- linux {
--    executable.path = /usr/local/bin
-+    executable.path = $$(out)/bin
-     executable.files = antimony
--    nodes_folder.path = /usr/local/bin/sb/nodes
-+    nodes_folder.path = $$(out)/bin/sb/nodes
-     nodes_folder.files = ../py/nodes/*
--    fab_folder.path = /usr/local/bin/sb/fab
-+    fab_folder.path = $$(out)/bin/sb/fab
-     fab_folder.files = ../py/fab/*
-     INSTALLS += executable nodes_folder fab_folder
- }
-diff --git a/qt/fab.pri b/qt/fab.pri
-index a54813b..b500536 100644
---- a/qt/fab.pri
-+++ b/qt/fab.pri
-@@ -54,7 +54,7 @@ DEFINES += '_STATIC_= '
- 
- linux {
-     QMAKE_CFLAGS += -std=gnu99
--    QMAKE_CXXFLAGS += $$system(/usr/bin/python3-config --includes)
-+    QMAKE_CXXFLAGS += $$system(python3-config --includes)
-     LIBS += -lpng
- }
- 
-diff --git a/qt/shared.pri b/qt/shared.pri
-index e7d0e3a..026eae3 100644
---- a/qt/shared.pri
-+++ b/qt/shared.pri
-@@ -39,41 +39,11 @@ macx {
- }
- 
- linux {
--    QMAKE_CXXFLAGS += $$system(/usr/bin/python3-config --includes)
--    QMAKE_LFLAGS   += $$system(/usr/bin/python3-config --ldflags)
-+    QMAKE_CXXFLAGS += $$system(python3-config --includes)
-+    QMAKE_LFLAGS   += $$system(python3-config --ldflags)
- 
-     # Even though this is in QMAKE_LFLAGS, the linker is picky about
-     # library ordering (so it needs to be here too).
-     LIBS += -lpython3.4m
--
--    # ldconfig is being used to find libboost_python, but it's in a different
--    # place in different distros (and is not in the default $PATH on Debian).
--    # First, check to see if it's on the default $PATH.
--    system(which ldconfig > /dev/null) {
--        LDCONFIG_BIN = "ldconfig"
--    }
--    # If that failed, then search for it in its usual places.
--    isEmpty(LDCONFIG_BIN) {
--        for(p, $$list(/sbin/ldconfig /usr/bin/ldconfig)) {
--            exists($$p): LDCONFIG_BIN = $$p
--        }
--    }
--    # If that search failed too, then exit with an error.
--    isEmpty(LDCONFIG_BIN) {
--        error("Could not find ldconfig!")
--    }
--
--    # Check for different boost::python naming schemes
--    LDCONFIG_OUT = $$system($$LDCONFIG_BIN -p|grep python)
--    for (b, $$list(boost_python-py34 boost_python3)) {
--        contains(LDCONFIG_OUT, "lib$${b}.so") {
--            LIBS += "-l$$b"
--            GOT_BOOST_PYTHON = True
--        }
--    }
+diff --git a/app/CMakeLists.txt b/app/CMakeLists.txt
+index ddc5c9b..d80728a 100644
+--- a/app/CMakeLists.txt
++++ b/app/CMakeLists.txt
+@@ -140,11 +140,11 @@ else()
+             ${CMAKE_CURRENT_BINARY_DIR}/sb/fab)
+ 
+     install(TARGETS antimony
+-            DESTINATION /usr/local/bin)
++            DESTINATION bin)
+     install(DIRECTORY ../py/nodes
+-            DESTINATION /usr/local/share/antimony)
++            DESTINATION bin/sb)
```

Comment:
What are those? It's not conventional to have non-executable files in a `/bin` subdirectory. Maybe move them to `/lib/antimony` or something?